app/assets/stylesheets/layout/_top_menu.sass

```diff
@@ -51,37 +51,51 @@
   %absolute-layout-mode &
     position: absolute
 
-  #search
-    margin: 5px 25px 0 0
+  #top-menu-search
     float: left
-  #search_wrap
+  .search-wrapper
     position: relative
-    top: 6px
+  .search_btn
+    display: block
+    height: $header-height
+    line-height: $header-height
+    @include default-font($header-item-font-color, $header-item-font-size)
+    &:hover
+      text-decoration: none
+      background: $header-item-bg-hover-color
+      color: $header-item-font-hover-color
   .search_field
-    border: $header-search-field-border
-    border-radius: 25px
-    width: 180px
-    outline: 0
-    padding: 7px 30px 7px 10px
-    font-size: 0.75rem
-    color: $header-search-field-font-color
-  ::-webkit-input-placeholder
-    color: $header-search-field-font-color
-  :-moz-placeholder
-    color: $header-search-field-font-color
-  span#magnifier
-    @include icon-common
-    position: absolute
-    top:   9px
-    right: 9px
-    &:after
+    display: none
```

Comment:
Ah, I see. That makes sense. In this case I agree that the inversion would be the right thing to do, so that without any modifiers the search is visible.

Also after reading the rules provided by @myabc it should probably be `-collapsed` instead of `search-collapsed`, which seems to make sense...